manifests/application.pvn.yaml

```
@@ -29,6 +29,14 @@ application:
           lifecycle:
             - postDeployment:
                 checkDuration: 600s
+    - name: test_templates
+      preconditions:
+        - releaseChannelStable:
+          releaseChannel: canary
+      runtimes:
+        - name: config-raw
+          runtime: marine-cycle-160323--govalctl-config-raw
+          type: EXTENSION
```

if you want to run k8s we dont need the custom runtime
```suggestion
      runtimes:
          - runtime: marine-cycle-160323--goval
          containerOrchestration:
            k8s:
              namespace: ansible
```